### ISSUE TITLE
Actually compiling the source code

### ISSUE THREAD

**Issue by Jhovall** [daid/EmptyEpsilon#17]
Hey,
We are very interested in your code/game and I am trying to compile the code. However, I have some problems actually getting the *.exe to work. Could you assist me?

I downloaded codeblocks (With minggw) and SFML 2-2: GCC 4.9.2 MinGW (DW2) - 32-bit.
I also downloaded the latest version of empty epsilon and serious proton and to my knowledge I assigned the linker and compilers settings in codeblocks correctly. (as described in the tutorial)
All 3 maps are in one (main) map.

Compiling gives no error, however, when starting up the empty epsilon.exe I get library errors. When I put those libraries in the *.exe-map it understands, however I get a procedure entry point error (or something).

Could you please assist me :)

**Comment by daid** [comment]
If you're getting entry point something errors, then most likely the version of SFML you are using does not match the Mingw version you are using.

**Comment by kwadroke** [comment]
I had to use TDM (SJLJ) version. For some reason the Release build didn't work, but, if I remember right, the debug one did.

**Comment by Jhovall** [comment]
Hey,
Thanks for the reply.
codeblocks-13.12mingw-setup.exe (for windows 7). Is the codeblock version I use, it is the newest version available. 
I don't see how I could make an error here :S.

But I will use the TDM version of codeblocks tonight. :)

**Comment by ErikDeBruijn** [comment]
FYI: http://software.ultimaker.com/EmptyEpsilon/

There's a release that was compiled today and will be likely updated quite frequently. Over the last couple of months that was weekly or more often.

Mac OS X version seems to be missing the binary but it will run fine with `wine EmptyEpsilon.exe` (wine-1.6.2)

**Comment by kwadroke** [comment]
@Jhovall I don't think there is a TDM version of codeblocks. I was referring to the TDM version of SFML

**Comment by Jhovall** [comment]
The solution was quite easy :(. You had to put the libraries of mingw near the *.exe, now it works

**Comment by kwadroke** [comment]
Yeah, that too :)

**Comment by Jhovall** [comment]
Well...

Yesterday compiling worked... untill I changed something in the code. (I tried to add another model pack). And the release didn't start.
Then I reversed the change I made (by downloading the lastest clone). 
The debug version runs normally, except for the fact that pressing the esc button closes the game.
The release version does not start up at all, no errors, it just doesn't start up.

Has this error seen before?
